### ISSUE TITLE
[Android] Live Photos / Motion Photos / Top Shot photos not working

### ISSUE THREAD

**Issue by null-von-sushi** [ente-io/photos-app#1551]
**Describe the bug**
Searching through bug reports, it seems that photos that include a few frames of video should be supported, however these photos are rendered as bog-standard static images

**To Reproduce**
0. Use Android, with access to Google Camera
1. Enable 'top shot' in the Google Camera Settings, then take a photo. Ensure that it's a live photo by checking in the Google Photos app (it should let you play the previous and next frames). 
2. Upload this image to Ente.
4. View the image either in the Ente app, or using the Desktop client (tested using Android and Linux (AppImage)).
6. Observe the lack of motion.

**Expected behavior**
The previous few frames before the image was taken should play, or the UI should have an obvious way to play these back (either via icon, or by tapping-and-holding the image itself (similar to the UX of Google Photos itself)).

**Screenshots**
Not necessary, I believe. Please correct me if I am wrong, and I will happily record a screencast or take a screenshot.

**Smartphone (please complete the following information):**
 - Device: Pixel 6 Pro
 - OS: Graphene OS (based on Android 14)
 - App Version 0.8.4 (latest from Google Play Store, at the time of filing this bug)

**Additional context**
I have the sneaky suspicion that this is less of a technical issue, and more of a 'anything except Apple's format is not implemented yet'.

As far as I am aware, there are only 3 formats for live photos. Whatever Apple does, `.MP.JPG` files created by Google Camera, and whatever Samsung uses. 

For your convenience, I have uploaded a zip file that has a `.MP.jpg` file in it. If viewed on the Google Photos app (Android), it will show the other frames. 
[PXL_20231128_142911998.MP.jpg.zip](https://github.com/ente-io/photos-app/files/13515005/PXL_20231128_142911998.MP.jpg.zip)

Also, I found [this](https://linuxreviews.org/Google_Pixel_%22Motion_Photo%22) which seems to be a technical analasys of the format itself.

**Comment by null-von-sushi** [comment]
I...am not sure why, but it magically started working for some photos (some work, some don't). 
For what it's worth, I just made my account and it was still in the process of syncing. Maybe it takes a while to generate the motion photo information?

**Comment by ua741** [comment]
Thank you for sharing the sample. We have identified the bug. 
Our motion photo detector has a check that assumed the video size will be greater than (original image size/2). 

This issue should be fixed in the next version. For the photos that are already uploaded, you will be able to long press and play the video. The app will also mark it as motion photo for future.

PS: Is it okay if we include this sample in our public [test data](https://github.com/ente-io/motion_photos/tree/main/assets) set for our plugin that detects motion photo? I have verified that this sample image doesn't contain any GPS co-ordindate.
